src/20240421_Multi_AprilFinanceUpdate/AprilFinanceUpdate.md

```diff
@@ -0,0 +1,89 @@
+
+Also within this proposal, there is the corective action of returning [3504 RealT WXDAI](https://gnosisscan.io/tx/0xf70a507847da2ad4acbecc35cd84c2f5d2489b0b0eb0e16af49c9262ca95707e) sent by RealT to the Aave Gnosis Chain collector by accident. Details on the agreement with realT can be found [here](https://snapshot.org/#/aave.eth/proposal/0xff69be7580614ebc1a455591c1bd651d8f0af12070d277d7d8846beb3c7c964b).
+
+This proposal will follow the Direct-To-AIP as outlined in this [proposal](https://governance.aave.com/t/arfc-funding-update/16675).
+
+Certain permissionless actions on the aave governance requires interaction (transaction) by someone for the governance system to run. This is done by the Aave Robot, which spends LINK tokens for the gas amount spent during the automation of these permissionless actions.
+
+During the past months BGD labs had spent in total of 1640 LINK for the Aave Robot, and 4000 MATIC for funding the Aave Delivery Infrastructure which is used to pay for the bridging fees in order for the governance system to work.
+
+BGD has paid the following amounts for security reviews so reimbursement is in order:
+
+- Compensation to Mixbytes for the review of Governance v3 Tokens: $30'000
+- Compensation to Emanuele Ricci (@stermi) for the review done on Aave 3.1 Feature: $12'000
+- Engagement with Spearbit for Aave v3 Ad-hoc Review: $109'200
+
+## Specification
+
+## BGD Reimbursements
+
+- Transfer 42'000 aUSDC v2 Ethereum to `0xb812d0944f8F581DfAA3a93Dda0d22EcEf51A9CF`
+- Transfer 109'200 aUSDT v2 Ethereum to `0xb812d0944f8F581DfAA3a93Dda0d22EcEf51A9CF`
+- Transfer 1640 aLINK v2 Ethereum to `0xb812d0944f8F581DfAA3a93Dda0d22EcEf51A9CF`
+- Transfer 4000 aMATIC v2 Polygon to `0xbCEB4f363f2666E2E8E430806F37e97C405c130b`
+
+### Migrating AGD's allowance to GHO
+
+The following Allowance is to be cancelled and replaced with GHO:
+
+- Cancel aUSDT Allowance
+- Increase GHO Allowance by 613,000 units
+
+The AGD mulisig `eth:0x89C51828427F70D77875C6747759fB17Ba10Ceb0` will be able to use the allowance function on `0x464C71f6c2F760DdA6093dCB91C24c39e5d6e18c` to claim the GHO.
+
+### Returning RealT's supplied WXDAI to RealT
+
+Transfer 3,504 armmv3WXDAI to `0x7DA9A33d15413F499299687cC9d81DE84684E28E` on Gnosis Chain.
+
+### Consolidate GHO funding and Migrate funds from V2 to V3
+
+#### Part A
+
+Perform the following Swaps, Migrate Eth v2 to v3 and bridge from Polyon to Ethereum.
+
+| Withdraw & Swap to GHO | Migrate Eth v2 to v3 |    Pol to Eth     | Migrate Pol v2 to v3 |
+| :--------------------: | :------------------: | :---------------: | :------------------: |
+|    aEthUSDC (1.4M)     |    awBTC (All-1)     |  aPolDAI (All-1)  |    amUSDT (All-1)    |
+|      aDAI (All-1)      |    awETH (All-1)     |  amUSDC (All-1)   |                      |
```

Comment:
I would put an approximate amount instead of (All - 1)

Reply:
we've been using this terminology in all these proposals, if it's okay i'd rather keep this way.

Reply:
okay